chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -321,32 +331,35 @@ Zotero.QuickCopy = new function() {
 				}
 				
 				var content = {
-					text: contentType == "html" ? html : text,
+					text: format.contentType == "html" ? html : text,
 					html: copyHTML
 				};
 				
 				return content;
 			}
 			
+			// determine locale preference
+			var locale = format.locale ? format.locale : Zotero.Prefs.get('export.quickCopy.locale');
+			
```

Comment:
Is that necessary? The value of extensions.zotero.export.bibliographyLocale wasn't validated either, right? What would be the validation criteria? Checking that the locale code value is a valid RFC 1766 locale tag? What would be returned if validation fails? An empty string?